views/error.hbs

```diff
@@ -0,0 +1,6 @@
+{{>header}}
+{{>calendar}}
```

Comment:
Calendar is probably unecessary for an error view. If the error is to do with the events API not responding then it shouldn't display a broken calendar.